packages/graph/hash_graph/lib/graph/src/types/schema/entity_type/mod.rs

```diff
@@ -1,11 +1,15 @@
+mod links;
+
 use std::collections::{HashMap, HashSet};
 
 use error_stack::{ensure, Result};
+// I do not know why rustfmt insists to put this import here despite the fact that it's not an
+// external crate
+use links::{Links, ValueOrMaybeOrderedArray};
```

Comment:
I guess I could fix this by using `crate::types::schema::entity_type::links` but that seems kinda dumb considering links is a submodule of this. 

If we don't really care about the ordering here I'll just delete the comment :)

Reply:
```suggestion
use self::links::{Links, ValueOrMaybeOrderedArray};
```

Reply:
insert big brain meme here